cli/src/commands/init/index.ts

```diff
@@ -209,9 +215,10 @@ export default class Init extends BaseCommand<typeof Init> {
       return { name: 'yarn', args: ['add'] };
     } else if (await this.access('pnpm-lock.yaml')) {
       return { name: 'pnpm', args: ['add'] };
-    } else {
-      return null;
+    } else if (await this.access('package.json')) {
+      return { name: 'npm', args: ['install', '--save'] };
```

Comment:
I'm not sure of this change, if you don't have a package manager configured I wouldn't force it to npm.

Can we add a new question at this stage? This should barely happen and I would like to avoid that we do unwanted changes to a users codebase.

Reply:
So it happened to me yesterday, like this: `pnpm init && xata init`

`pnpm init` doesn't create a lock file, only a `package.json`.

So I think for people who are just trying us out this might be more common than we think.

Reply:
@SferaDev I've backed out this change in this PR, will follow up with a a PR to add an extra question for this code path 👍